playerbot/strategy/generic/CombatStrategy.cpp

```diff
@@ -7,10 +7,6 @@ using namespace ai;
 
 void CombatStrategy::InitTriggers(list<TriggerNode*> &triggers)
 {
-    triggers.push_back(new TriggerNode(
-        "enemy out of spell",
-        NextAction::array(0, new NextAction("reach spell", 60.0f), NULL)));
-

```

Comment:
Removing this from the default combat strategy (which is also used by healer bots) and move it to caster bot strategies